### ISSUE TITLE
Non-arm64e RootLess support

### ISSUE THREAD

**Issue by amania-Jailbreak** [ryannair05/Cylinder-Reborn#20]
I am in a Palera1n RootLess environment on an iPhone 6s
My question is, currently only arm64e devices support RootLess, are there any plans to support other devices?
Rootful is fine, but I use RootLess to avoid detection, so it would be great if you could support arm64-based devices other than arm64e!

![image](https://github.com/ryannair05/Cylinder-Reborn/assets/91243704/60885201-b2a5-4362-bac2-bba330902d7d)

```
Sorry, but some sentences may be incorrect due to the use of Deepl translation.
```

**Comment by TrippinEnvy** [comment]
I agree with Amanda. It would be nice for a update to support 6s plus devices running  on palera1n

**Comment by kolbicz** [comment]
I compiled an ARM64 Rootless version. you can get it here: https://github.com/kolbicz/Cylinder-Reborn/releases/tag/ARM64
